Tests.Integration.Presentation.Web/SystemUsage/V2/ItSystemUsageApiV2Test.cs

```diff
@@ -747,20 +750,129 @@ private static async Task AssertOrganizationalUsage(string token, Guid systemUsa
+
+            //Act
+            var createdDTO = await ItSystemUsageV2Helper.PostAsync(token, CreatePostRequest(organization.Uuid, system.Uuid, roles: roles));
+
+            //Assert
+            var freshReadDTO = await ItSystemUsageV2Helper.GetSingleAsync(token, createdDTO.Uuid);
+            Assert.Equal(2, freshReadDTO.Roles.Count());
+            var right1 = Assert.Single(freshReadDTO.Roles.Where(x => x.User.Uuid == user1.Uuid));
+            Assert.Equal(role.Name, right1.Role.Name);
+            Assert.Equal(role.Uuid, right1.Role.Uuid);
+            Assert.Equal(user1.GetFullName(), right1.User.Name);
+            var right2 = Assert.Single(freshReadDTO.Roles.Where(x => x.User.Uuid == user2.Uuid));
+            Assert.Equal(role.Name, right2.Role.Name);
+            Assert.Equal(role.Uuid, right2.Role.Uuid);
+            Assert.Equal(user2.GetFullName(), right2.User.Name);
+        }
+
+        [Fact]
+        public async Task Can_PUT_Modify_Roles()
+        {
+            //Arrange
+            var organization = await CreateOrganizationAsync(A<OrganizationTypeKeys>());
+            var (user, token) = await CreateApiUser(organization);
+            await HttpApi.SendAssignRoleToUserAsync(user.Id, OrganizationRole.LocalAdmin, organization.Id).DisposeAsync();
+            var system = await CreateSystemAndGetAsync(organization.Id, AccessModifier.Public);
+
+            var user1 = await CreateUser(organization);
+            var user2 = await CreateUser(organization);
+            var role = DatabaseAccess.MapFromEntitySet<ItSystemRole, ItSystemRole>(x => x.AsQueryable().First());
+
+            var createdDTO = await ItSystemUsageV2Helper.PostAsync(token, CreatePostRequest(organization.Uuid, system.Uuid));
+
+            var initialRoles = new List<RoleAssignmentRequestDTO>
+            {
+                new RoleAssignmentRequestDTO()
+                {
+                    RoleUuid = role.Uuid,
+                    UserUuid = user1.Uuid
+                }
+            };
+            var modifyRoles = new List<RoleAssignmentRequestDTO>
+            {
+                new RoleAssignmentRequestDTO()
+                {
+                    RoleUuid = role.Uuid,
+                    UserUuid = user2.Uuid
+                }
+            };
+
+            //Act - Add role
+            var addInitialRolesRequest = await ItSystemUsageV2Helper.SendPutRoles(token, createdDTO.Uuid, initialRoles);
+
+            //Assert
+            Assert.Equal(HttpStatusCode.OK, addInitialRolesRequest.StatusCode);
+            var initialRoleResponse = await ItSystemUsageV2Helper.GetSingleAsync(token, createdDTO.Uuid);
+            var initialRight = Assert.Single(initialRoleResponse.Roles);
+            Assert.Equal(role.Name, initialRight.Role.Name);
+            Assert.Equal(role.Uuid, initialRight.Role.Uuid);
+            Assert.Equal(user1.Uuid, initialRight.User.Uuid);
+            Assert.Equal(user1.GetFullName(), initialRight.User.Name);
+
```

Comment:
Du mangler at bevise at din deltaberegning virker (det gør den ikke lige nu for du har ingen equality)